inspirehep/modules/fixtures/users.py

```diff
@@ -46,5 +46,19 @@ def init_users_and_permissions():
             action='superuser-access',
             role=superuser_role
         ))
-        db.session.add(ActionRoles(action='admin-access', role=superuser_role))
+        db.session.add(ActionRoles(
+            action='admin-access',
+            role=superuser_role)
+        )
+
+        # Create a user that has access to restricted collection
+        restricted_user = ds.create_user(
```

Comment:
This is just for testing at the moment, will add this user and permission to a test instead.